examples/Receiver_Response.py

```diff
@@ -0,0 +1,245 @@
+    :math:`μV(\\nu)` which is fit to an absolute sinc function.
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+from numpy import r_, pi
+import pyspecdata as psd
+from sympy import symbols
+import sympy as sp
+import re
+
+Dnu_name = r"$\Delta\nu$"
+nu_direct = r"$\nu_{direct}$"
+lambda_G = 0.4e3  # Width for Gaussian convolution
+data_dir = "ODNP_NMR_comp/noise_tests"
+file1 = "240123_10mV_AFG_GDS_5mV_100MSPS_analytic.h5"
+file2 = "240117_afg_sc_10mV_3p9kHz_zoom.h5"
+
+
+# {{{ Function to return the nodenames, test signal frequencies, and an
+#     empty nddata to drop calculated dg and V into
+def make_ndData(thisfile, nodes_in_kHz=True):
+    """Determine the nodenames of the file and sort them according to the
+    frequency of the test signal. Based on these nodenames a list of the
+    frequencies as integers are made. Anempty NDData in the shape of the 
+    acquired data is created with an extra dimension 'nu_test' that 
+    the sorted frequencies are placed inside that dimension.
+
+    Parameters
+    ==========
+    thisfile: str
+        Name of file that has nodes containing test signal at varying
+        frequencies.
+    nodes_in_kHz: bool
+        Typically the nodenames are saved with the signal frequency
+        in kHz. However, in older files the nodenames are saved with
+        the signal frequency in Hz. If the frequencies are in Hz and
+        this argument is true, the frequencies will be converted to
+        Hz.
+
+    Returns
+    =======
+    nodenames: list
+        List of strings of the nodenames sorted by the test signal frequency.
+    these_frqs: list
+        List of sorted signal frequencies as integers
+    ret_data: nddata
+        Empty nddata in the appropriate shape with an extra 'nu_test'
+        dimension that contains the signal frequencies
+    """
+    nodenames = sorted(
+        psd.find_file(
+            re.escape(thisfile),
+            exp_type=data_dir,
+            return_list=True,
+        ),
+        key=lambda x: int(x.split("_")[1]),
+    )
+    if nodes_in_kHz:
+        these_frqs = (
+            np.array(
+                [int(j.split("_")[1]) for j in nodenames]
+            )
+            * 1e3
+        )
+    else:
+        these_frqs = np.array(
+            [int(j.split("_")[1]) for j in nodenames]
+        )
+    # Determine if nScans is a dimension in the acquired data
+    # If nScans is a dimension this indicates the data was acquired on the
+    # receiver with the intention to convert to a PSD in which case we will
+    # just pull the shape of one of the datasets and copy its shape first
+    og_data = psd.find_file(
+        thisfile, expno=nodenames[0], exp_type=data_dir
+    )
+    if "nScans" in og_data.dimlabels:
+        ret_data = (
+            og_data.shape + ("nu_test", len(these_frqs))
+        ).alloc()
+        ret_data.setaxis(
+            "t", og_data.getaxis("t")
+        ).set_units("t", "s")
```

Comment:
decided to keep this as t rather than label nu direct so it makes more sense when we calculate the psd first (acq time)

Reply:
why do you have something called og_data -- that does not seem right at al.

Reply:
OK, in context, I see